client/apps/settings/client/curations/display/components/canvas/index.jsx

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import { CanvasControls } from './canvas_controls.jsx'
+import { CanvasImages } from './canvas_images.jsx'
+import { CanvasText } from './canvas_text.jsx'
+
+export class Canvas extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      activeLayout: props.campaign.canvas.layout || 'overlay'
+    }
+  }
+
+  setActiveLayout = (layout) => {
+    this.setState({activeLayout: layout})
```

Comment:
We should get prettier wired up here, but typically spaces are left between objects `this.setState({ activeLayout: layout })`

Reply:
For now this can be checked by `.eslintrc`. 

Speaking of Prettier https://github.com/prettier/prettier/issues/3101. IMO, it ruins JSX by making it sloppy and unreadable. I wish there was an option to disable checks in those areas.